src/borg/archiver.py

```diff
@@ -2348,8 +2348,8 @@ def do_break_lock(self, args, repository):
             /home/*/junk
             *.tmp
             fm:aa:something/*
-            re:^home/[^/]\\.tmp/
-            sh:home/*/.thumbnails
+            re:^/home/[^/]+\\.tmp/
+            sh:/home/*/.thumbnails
```

Comment:
btw, for this PR it is sufficient to change these 2 lines.

did you do the other changes manually or did you run python setup.py build_man/build_usage?
it is not needed to do that, i run these at release time.

Reply:
I did it manually. Would you like me to revert the changes?

Reply:
yeah, reverting the 2 unneeded changes would be good. thanks!